Index: damnvid/locale/Spanish.locale
===================================================================
--- damnvid.orig/locale/Spanish.locale	2010-05-26 01:41:32.000000000 -0400
+++ damnvid/locale/Spanish.locale	2010-05-26 01:41:39.000000000 -0400
@@ -424,7 +424,7 @@
 		'The video history is empty.':u'El historial de vídeo está vacío.',
 		'Adds a damn video from the video history.':u'Añade un maldito vídeo desde el historial.',
 		'Show DamnVid':u'Mostrar DamnVid',
-		'Perform search when launched':u'Hacer una búsqueda nada más arrancar'
+		'Perform search when launched':u'Hacer una búsqueda nada más arrancar',
 		'Reset status':u'Reiniciar estado',
 		'QuickTime format (.mov)':u'Formato QuickTime (.mov)',
 		'Proxy':u'Proxy',
