Index: damnvid/conf/conf.ini
===================================================================
--- damnvid.orig/conf/conf.ini	2010-05-05 22:04:11.470119688 +0200
+++ damnvid/conf/conf.ini	2010-05-05 22:06:58.970123879 +0200
@@ -5,7 +5,7 @@
 lastprefdir =
 lastmoduledir =
 threads =
-checkforupdates = True
+checkforupdates = False
 defaultoutdir = ?DAMNVID_MY_VIDEOS?/DamnVid/
 clipboard = True
 autoconvert = False
@@ -311,4 +311,4 @@
 encoding_qmax = 31
 encoding_f = mp4
 encoding_g =
-encoding_b = sameq
\ No newline at end of file
+encoding_b = sameq
Index: damnvid/conf/preferences.damnvid
===================================================================
--- damnvid.orig/conf/preferences.damnvid	2010-05-05 22:09:47.762118570 +0200
+++ damnvid/conf/preferences.damnvid	2010-05-05 22:10:13.000000000 +0200
@@ -280,7 +280,7 @@
 		'kind':'bool',
 		'strict':True,
 		'align':False,
-		'default':True
+		'default':False
 	},
 	'damnvid:defaultoutdir':{
 		'name':'Default output directory',
