Index: damnvid/DamnVid.py
===================================================================
--- damnvid.orig/DamnVid.py	2010-05-06 23:15:59.974124157 +0200
+++ damnvid/DamnVid.py	2010-05-06 23:20:51.000000000 +0200
@@ -3501,10 +3501,8 @@
 					os_exe_ext = '.exe'
 				elif DV.os == 'mac':
 					os_exe_ext = 'osx'
-				if DV.bit64:
-					os_exe_ext = '64' + os_exe_ext
 				self.passes = 1
-				cmd = [DV.bin_path + 'ffmpeg' + os_exe_ext, '-i', '?DAMNVID_VIDEO_STREAM?', '-y', '-passlogfile', DV.tmp_path + 'pass']
+				cmd = ['/usr/bin/ffmpeg' + os_exe_ext, '-i', '?DAMNVID_VIDEO_STREAM?', '-y', '-passlogfile', DV.tmp_path + 'pass']
 				for i in DV.preferences.keys():
 					if i[0:25] == 'damnvid-profile:encoding_':
 						i = i[16:]
