Index: damnvid/DamnVid.py
===================================================================
--- damnvid.orig/DamnVid.py	2010-06-02 18:51:08.000000000 -0400
+++ damnvid/DamnVid.py	2010-06-02 18:52:12.000000000 -0400
@@ -1181,8 +1181,10 @@
 	def __init__(self, func, *args, **kwargs):
 		self.func = func
 		self.pending = args[:]
-		self.kwargs = kwargs
+		self.kwargs = kwargs.copy()
+		Damnlog('DamnCurry initiated with function', func, 'and curry arguments:', self.pending, '; keyword arguments:', self.kwargs)
 	def __call__(self, *args, **kwargs):
+		Damnlog('DamnCurry for function', self.func, 'called with extra arguments:', args, 'and extra keyword args:', kwargs)
 		if kwargs and self.kwargs:
 			kw = self.kwargs.copy()
 			kw.update(kwargs)
@@ -2366,6 +2368,9 @@
 	def onLoad(self, event):
 		info = event.GetInfo()
 		Damnlog('onLoad event on YouTube browser. Event data:', info)
+		if info.has_key('newsearch'):
+			self.onSearchPostEvent(info['newsearch'], event)
+			return
 		if info['query'][0] == 'feed':
 			results = info['result']
 			boldfont = wx.SystemSettings.GetFont(wx.SYS_DEFAULT_GUI_FONT)
@@ -2520,8 +2525,11 @@
 			self.Bind(wx.EVT_MENU, DamnCurry(self.onSearchMenu, i), item)
 		Damnlog('History menu built, assigning it to search box.')
 		self.searchbox.SetMenu(boxmenu)
-	def onSearchMenu(self, query, event):
-		Damnlog('YouTube browser dialog received onSearchmenu event. Query is', query)
+	def onSearchMenu(self, query, event=None):
+		Damnlog('YouTube browser dialog received onSearchMenu event. Query is', query)
+		DamnPostEvent(self, DamnLoadingEvent(DV.evt_loading, -1, {'newsearch':query})) # Fix wxPython on Linux bug: Need to be in a new event loop iteration to do the search (and potentially, menu rebuilding).
+	def onSearchPostEvent(self, query, event=None):
+		Damnlog('YouTube browser dialog received onSearchPostEvent event. Query is', query)
 		if query is None: # Clear history
 			DV.prefs.seta('damnvid-search', 'history', [])
 			DV.prefs.save()
